homeworks/rostyslav.havuka_RostyslavHavuka/5-rxjs/main.js

```diff
@@ -0,0 +1,20 @@
+const { fromEvent, animationFrameScheduler } = window.rxjs;
+const {
+    pairwise, map, distinctUntilChanged, throttleTime,
```

Comment:
(not blocking). I would better wrap all long imports on the next line each:

```js
const {
    pairwise,
    map,
    distinctUntilChanged,
    throttleTime,
} = window.rxjs.operators;
```

Code will be more readable and more scallable 😄